lib/github_service.rb

```diff
@@ -113,6 +113,21 @@ def refresh_assignees(fq_name)
       assignees_cache.delete(fq_name)
     end
 
+    def username_lookup(username)
+      if username_lookup_cache.key?(username)
+        username_lookup_cache[username]
+      else
+        username_lookup_cache[username] ||= begin
+          case Net::HTTP.get_response(URI("https://github.com/#{username}"))
```

Comment:
I feel like an octokit lookup would be cheaper here than a full http get of their profile.

Reply:
Alternately a HEAD request would be cheap as well and would still give you response 200/404

Reply:
Good idea, will change.